templates/minimal/README.md

```diff
@@ -37,21 +37,6 @@ A Polkadot SDK based project such as this one consists of:
 * 🛠️ Depending on your operating system and Rust version, there might be additional
 packages required to compile this template - please take note of the Rust compiler output.
 
-## Customization
-
-In case you would like to change the pallet's name from  `pallet-minimal-template` to `pallet-test`,
-you would need to implement the following changes:
```

Comment:
I think this was recently added https://github.com/paritytech/polkadot-sdk/pull/4806

Reply:
Yes, but I had a bit of disagreement with it 🙈  lmk if you think I am being too harsh here: https://github.com/paritytech/polkadot-sdk/pull/4806#issuecomment-2203372115

Reply:
Hm not sure. But yea could be that it is too low level.